README.md

```diff
@@ -36,8 +36,10 @@ people for programming help in DMs.
    nicknames).  Do not post or discuss piracy content. Do not post or discuss
    political content.
 
-9. No advertisement (invite links, referral links, etc.) unless explicitly asked,
-   in which case links can be sent through personal messages.
+9. No advertisement (commercial products, referral links, etc.) unless
+   explicitly asked, in which case links can be sent through personal messages.
```

Comment:
We agreed that they could share links in server if asked for, if I recall.

Reply:
There is an explicit allowance about invite links, do we make the rule apply to everything?

The guideline that was voted on:
```
2. Invite links are allowed in the server if asked for. Jim auto removes links, users can ask a mod to put their link since it won't be picked up by Jim. Links that escaped Jim will be removed if they are spam.
```

Reply:
Oh, I see that you removed invite links from the first part. I think we could extend it to everything and see how it goes, and pull back if it becomes a problem later.

Reply:
Hm, something like this?

```
No advertisement (invite links, referral links, commercial products, etc.) unless explicitly asked, in which case links can be sent through the server. Discord server invite links will be removed automatically by Safety Jim for anti-spam measures, you must ask a moderator to share your server for you.
```